client/my-sites/plans/jetpack-plans/product-grid/utils.ts

```diff
@@ -34,12 +34,7 @@ export const getPlansToDisplay = ( {
 				product.term === duration &&
 				// Don't include a plan the user already owns, regardless of the term
 				! currentPlanTerms.includes( product.productSlug )
-		)
```

Comment:
`getJetpackDescriptionWithOptions` actually now just returns the product description.